doc/tutorials/access_store_ui.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.            def __getattr__(self, key):
</pre><p>That should be already available in the notebook connector. @chunki is currently releasing the newest version, then you can use it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>This has been merged to main but not yet released. So is the temporary hack.</p>

Reply:
<p>should be released now</p>